package.json

```diff
@@ -87,10 +86,10 @@
     "zustand": "^4.3.9"
   },
   "peerDependencies": {
-    "react": ">=18.12.0",
-    "react-dom": ">=18.12.0",
-    "snarkyjs": ">=0.12.1",
-    "zod": ">=3.21.4",
+    "react": ">=18.0.0",
+    "react-dom": ">=18.0.0",
+    "snarkyjs": ">=0.11.0",
```

Comment:
Why are we not using the latest version?

Reply:
peer dependencies are what the user should install in their project, not what we use to build. here we are saying that they should have at least those versions. wht we use to build is specified in dev deps